hutch_python/happi.py

```diff
@@ -1,33 +1,47 @@
 import inspect
 import logging
+from typing import Dict
 
 import happi
+import ophyd
 from happi.loader import load_devices
 
 try:
     import lightpath
+    from lightpath import LightController
     from lightpath.config import beamlines
 except ImportError:
     lightpath = None
+    LightController = None
     beamlines = None
 
 logger = logging.getLogger(__name__)
 
 
-def get_happi_objs(db, hutch):
```

Comment:
(We most likely didn't use this old function signature elsewhere, right? I doubt it...)

The new implementation makes a lot of sense and is a big improvement 👍